### ISSUE TITLE
Why did you rename fast_i18n?

### ISSUE THREAD

**Issue by mjablecnik** [Tienisto/slang#83]
Hello, I see that you renamed `fast_i18n` to `slang` but I don't know why?
What was wrong with previous name of this package?
Thank you for your response

**Comment by Tienisto** [comment]
Hi,
`fast_i18n` was originally created as a hobby project because I wanted a very fast and lightweight i18n solution.
`fast` was the first thing I came to mind. Over 2 years this library became more complex and the main focus transitioned from fast to type-safe. I actually disliked this name for quite some time but I hate breaking changes.

Then a user opened #61 proposing a federated package structure which makes sense if you want to develop dart only apps or if you want to use `fast_i18n` accross several packages in one project.
I found this change big enough to rename the current library.
